ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -614,3 +614,71 @@ def test_tensorflow_Minimum(
         x=np.asarray(x[0], dtype=input_dtype[0]),
         y=np.asarray(x[1], dtype=input_dtype[1]),
     )
+
+
+# Less
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(set(ivy_tf.valid_float_dtypes))
```

Comment:
Please use `ivy.valid_float_dtypes` as this lets us set the proper `dtypes`, like here: https://github.com/unifyai/ivy/blob/3d0cb56dd917cd6aaec6ce0e4e61f726d900ce47/ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py#L17

Reply:
Solved in https://github.com/unifyai/ivy/pull/3714/commits/33e92c4631aa6b6dfb4a94e162e303e1d0caa5fc